File: src/main/java/serendustry/machine/RecipeMapIndustrialLaboratory.java
package serendustry.machine;

import gregtech.api.recipes.RecipeMap;

public class RecipeMapIndustrialLaboratory extends RecipeMap<LaboratoryRecipeBuilder> {
    public RecipeMapIndustrialLaboratory(String name, int inputs, int outputs, int fluidInputs, int fluidOutputs, boolean hidden) {
        super(name, inputs, outputs, fluidInputs, fluidOutputs, new LaboratoryRecipeBuilder(), hidden);
    }
    // todo custom layout
}

File: src/main/kotlin/serendustry/machine/RecipeMapIndustrialLaboratory.kt
package serendustry.machine

import gregtech.api.recipes.RecipeMap

class RecipeMapIndustrialLaboratory(name: String, inputs: Int, outputs: Int, fluidInputs: Int, fluidOutputs: Int, hidden: Boolean):
    RecipeMap<LaboratoryRecipeBuilder>(name, inputs, outputs, fluidInputs, fluidOutputs, LaboratoryRecipeBuilder(), hidden) {

    // todo custom layout
}
